### DISCUSSION TITLE
Is it safe to gitignore the /videos folder?

### DISCUSSION THREAD

**Discussion by lpbonomi** [muxinc/next-video#199] [Q&A] [answered]
I wanted to gitignore the /videos folder, is it safe to do it?

**Comment by luwes** [comment]
it's not safe to ignore because the `*.json` video config files need to be read on the client side.
The raw video files (mp4's) should already be added to your gitignore if you used the `npx next-video init` command.

**Reply by lpbonomi** [reply]
Should I commit them then? What happens when there is more than one evironment (e.g dev, prod) that don't have the same videos?

**Reply by luwes** [reply, marked as answer]
yes, the json files should be committed to git if you want them to be available in that environment.
you could always make a subfolder and gitignore accordingly for dev and prod...

/videos/dev/
/videos/

**Reply by lpbonomi** [reply]
Ok, I see, it's a nice solution. The only thing is that I should periodically enter the production server and commit the files, which is not very convenient.

Have you thought about having an approach similar to next-auth that persists certain info in the db? https://authjs.dev/getting-started/adapters?_gl=1*1yaeya*_gcl_au*Mzg3ODM1NzUxLjE3MDk4NDAxMDk.

If you think this is a good idea I might be able to give a hand

**Reply by luwes** [reply]
yes, that's something we'd like to add!
here's a ticket for that https://github.com/muxinc/next-video/discussions/143

I had been looking into Drizzle ORM to get a first adapter in next-video but I can't work on it for a while, so if you like to contribute on this that would be much appreciated! make sure if you open a PR not to make it too big at once, small iterative changes please.

**Reply by lpbonomi** [reply]
Sounds good! I'll try to put in some work this week
